isis/src/control/apps/findfeatures/FeatureAlgorithmFactory.cpp

```diff
@@ -140,7 +140,7 @@ int FeatureAlgorithmFactory::initialize() {
   numAliases += m_algorithmInventory.addFeatureAlgorithm<ORBAlgorithm>();
   numAliases += m_algorithmInventory.addFeatureAlgorithm<SIFTAlgorithm>();
   numAliases += m_algorithmInventory.addFeatureAlgorithm<StarAlgorithm>();
-  numAliases += m_algorithmInventory.addFeatureAlgorithm<SURFAlgorithm>();
+  // numAliases += m_algorithmInventory.addFeatureAlgorithm<SURFAlgorithm>();
```

Comment:
Is SURF no longer supported?

Reply:
Correct, OpenCV moved SURF out and into an [OpenCV Contrib repo](https://github.com/opencv/opencv_contrib) where we would have to build OpenCV ourselves against the contrib library.

I'm guessing this constitutes an API breaking change :/

Reply:
I think this is likely API breaking.

At a minimum, we need to update the findfeatures docs so that SURF isn't listed in them.

Reply:
@jessemapel Should I simply remove them from the algorithm list or remove examples that imply you can use it as well?

Reply:
If I may suggest, perhaps there is way to retain all the code, trap the error and gracefully continue. This because if we (or other developers for that matter) can ever get SURF back, no change would need to be made to the code. findfeatures should rightfully throw an error if its requested and cannot find/invoke the algorithm.

Reply:
@KrisBecker So you're suggestion is leave this commented and let OpenCV throw it's error? Presumably we would still have to change the docs to remove the parameter?